internal/identity/identity.go

```diff
@@ -9,13 +9,20 @@ import (
 	"github.com/redhatinsights/platform-go-middlewares/identity"
 )
 
+type Principal = identity.XRHID
```

Comment:
This is the new type alias, everything else should remain the same except there is no need for users of these functions to import "github.com/redhatinsights/platform-go-middlewares/identity".